test/ButtonExtension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ButtonExtension.swift
+//  test
+//
+//  Created by apple on 05/06/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIButton
+{
```

Comment:
As a general convention when creating a new class, extension, struct etc. you shoud move the bracket on the declaration line with an empty space between and a empty line after